README.rst

```diff
@@ -24,6 +24,44 @@ Pyccel comes with a selection of **extensions** allowing you to convert calls to
 - mpi4py
 - h5py (not available yet)
 
+Pyccel Installation Methods
+***************************
+
+Pyccel can be installed / deployed in different ways:
+
+1. using Pypi
+2. using containers
+3. from source
```

Comment:
I find this list a bit confusing: Pyccel is always installed in one unique way, using `pip`.

The user can choose to install a version from PyPI, or the latest "trunk" version from GitHub, but this does not change anything really. In both cases `pip` will install the required Python dependencies. And in both cases we are installing the Python source files, not some pre-compiled binaries.

A separate discussion can be made for how to install the non-Python dependencies. So far we have assumed that the user can take care of the installations, by following step-by-step the simple instructions given below for the various operating systems. If we were to use `conda`, we could have it install the non-Python dependencies, but we are no conda experts and did not want to maintain that.

A Docker image bypasses all of that by including everything in the container, Pyccel + Python dependencies + non-Python dependencies + Linux operating system. But you still use `pip` to install Pyccel inside the Docker container.

So I think this should read more accurately:

> Pyccel can be installed on virtually any machine that provides Python 3, the `pip` package manager, a C/Fortran compiler, and an Internet connection. Some advanced features of Pyccel require additional non-Python libraries to be installed, for which we provide detailed instructions below.
>
> Alternatively, Pyccel can be deployed through a **Linux Docker image** that contains all dependencies, and which can be setup with any version of Pyccel.

Reply:
I can make this changes myself in this PR.

Reply:
Sorry, apparently I cannot. This PR comes from a fork of Pyccel. I will create a new PR instead.